v2/passwordless/updating-supertokens.mdx

```diff
@@ -0,0 +1,54 @@
+---
+id: updating-supertokens
+title: Updating Supertokens SDKs and Core
+hide_title: true
+---
+
+<!-- COPY DOCS -->
+<!-- ./community/updating-supertokens.mdx -->
+
+# Updating Supertokens SDKs and Core
+
+Not all older releases of every supertokens Core, front-end SDK, and back-end SDK are compatible with each other. In order to maintain compatability
+and keep up on new features, we recommend updating and upgrading all of your SDKs regularly, starting with the supertokens core, then the back-end SDK, then the front-end SDK.
+
+The following is a link to each core and SDK changelog, containing steps to migrate from each version. You can use our [compatability table](../../community/compatibility-table) to see which versions you need to upgrade to.
+
+## Core Migration
+
+- ### Self-hosted Core
+[supertokens-core](https://github.com/supertokens/supertokens-core/blob/master/CHANGELOG.md)
+
+- ### Managed Core
+If you're using the managed core service, please email us at team@supertokens.com from your registered email to request a core update.
+
+## Back-end SDK Migration
+
+- ### NodeJS
+[supertokens-node](https://github.com/supertokens/supertokens-node/blob/master/CHANGELOG.md)
```

Comment:
please dont just link to the changelog.. you need to tell users that this is the changelog and to look for the migration section under each version from the one they have uptil the one they want to move to.